qemu.mk

```diff
@@ -191,6 +191,11 @@ define run-help
 	@echo xtest 2001
 endef
 
+define launch-terminal
+	@nc -z  127.0.0.1 $(1) || \
```

Comment:
I like this, but can you please also update the [dependency-list](https://github.com/OP-TEE/optee_os#431-prerequisites) in optee_os to include netcat also?